### DISCUSSION TITLE
[2021/10/26] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#173] [General]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/issues/165 

 _Created from https://github.com/wantedly/frontend-night/issues/65 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by qnighy** [comment]
書きました: Rails 7.0でアセットパイプラインはどう変わるか https://www.wantedly.com/companies/wantedly/post_articles/354873

書きました: JavaScriptの非同期処理をじっくり理解する (4) AbortSignal, Event, Async Context https://zenn.dev/qnighy/articles/772f632af595aa

**Comment by prsdta** [comment]
## Kent C Dodds favorite tools
https://kentcdodds.com/blog/how-i-built-a-modern-website-in-2021#technology-overview

**Reply by prsdta** [reply]
始めてみたは https://mswjs.io/

**Comment by izumin5210** [comment]
## tsconfig の話をした


https://github.com/uhyo/rocon/blob/v1.2.3/tsconfig.json 見てなるほどって言った
